Flappy Bird Clone/Assets/Scripts/Player/PlayerBehaviour.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using Input;
+using ScriptableObjects;
+using UnityEngine;
+
+namespace Player
+{
+    public class PlayerBehaviour : MonoBehaviour
+    {
+        [SerializeField] private PlayerInfo playerInfo;
+
+        private InputController _inputController;
+
+        private SpriteRenderer _spriteRenderer;
+        private Rigidbody2D _physicsComponent;
+
+        private void Awake()
+        {
+            _inputController = FindFirstObjectByType<InputController>();
+            _physicsComponent = GetComponent<Rigidbody2D>();
+            _spriteRenderer = GetComponent<SpriteRenderer>();
+
+            if (!_inputController)
```

Comment:
Use `something != null` instead of `!something`. Read more here: https://github.com/JetBrains/resharper-unity/wiki/Possible-unintended-bypass-of-lifetime-check-of-underlying-Unity-engine-object